files/default/github_pr_comment_trigger.rb

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env ruby
+
+# Takes a JSON payload from a Github PR issue webhook from stdin, reads the
+# comment, and if it matches `!bump (major|minor|patch)?` (defaults to
+# `minor`), merges the PR, bumps the Chef cookbook's metadata.rb, creates a
+# version tag, and pushes up.
+
+require 'git'
+require 'json'
+require 'octokit'
+require 'pp'
+
+GIT_BRANCH = 'master'
+METADATA_FILE = 'metadata.rb'
+COMMAND = '!bump'
+LEVELS = {
+  'major' => 0,
+  'minor' => 1,
+  'patch' => 2
+}
+
+d = JSON.load(STDIN.read)
+
+# Check if we got a valid request and get the bump level
+comment = d.fetch('comment', {}).fetch('body', '')
+match = comment.match(/^#{COMMAND}( (#{LEVELS.keys.join('|')}))?$/, 2)
```

Comment:
It was to make both `!bump` and `!bump minor` (for example) valid.  One group contains the bump level, but I only want to match the bump level if there is a space in front of it (e.g. to prevent matching `!bumpminor`).

http://refiddle.com/refiddles/56983fc675622d3ff7a50000

Lines 1 and 3 should match.  Removing the outer group makes `!bump` require a trailing space.